src/testing/builder/NewsModelBuilder.js

```diff
@@ -38,10 +30,7 @@ class LocalNewsModelBuilder {
         newsItem: new LocalNewsModel({
           id: 12,
           title: 'first news item',
-          availableLanguages: new Map(difference(LANGUAGES, [this._language]).map(
-            lng => [lng, null]
-          )),
-          timestamp: new Date(),
+          timestamp: moment('2017-11-18 19:30:00', moment.ISO_8601),
```

Comment:
Please rather use the following:
```suggestion
          timestamp: moment('2017-11-18T19:30:00.000Z'),
```

Reply:
Still todo, right?

Reply:
You did it already, right?